frontend/src/app/testQueue/AoEForm/AoEModalForm.jsx

```diff
@@ -321,28 +322,38 @@ const AoEModalForm = ({
     onClose();
   };
 
-  const actionButtons = (
-    <div style={{ float: "right" }}>
-      <Anchor text="Cancel" onClick={onClose} />
+  const buttonGroup = (
+    <div className="sr-time-of-test-buttons" style={{ float: "right" }}>
+      <Button unstyled label="Cancel" onClick={onClose} />
       <Button label={saveButtonText} onClick={saveAnswers} />
     </div>
   );
 
   return (
-    <CMSDialog
-      onExit={onClose}
-      closeText="Cancel"
-      heading={displayFullName(
-        patient.firstName,
-        patient.middleName,
-        patient.lastName
-      )}
-      getApplicationNode={() => {
-        document.getElementById("#root");
+    <Modal
+      isOpen={true}
+      style={{
+        content: {
+          inset: "3em auto auto auto",
+          overflow: "auto",
+          maxHeight: "90vw",
+          width: "50%",
+          minWidth: "20em",
+          marginRight: "50%",
+          transform: "translate(50%, 0)",
+        },
       }}
-      actions={actionButtons}
-      actionsInHeader={true}
+      overlayClassName="prime-modal-overlay"
+      contentLabel="Time of Test Questions"
```

Comment:
This language and all its permutations would confuse me if I were using this application, but that may be systemic to healthcare right now...